dipy/reconst/tests/test_sfm.py

```diff
@@ -109,3 +109,29 @@ def fit(self, X, y):
                       sfm.SparseFascicleModel,
                       gtab,
                       solver=EvenSillierSolver())
+
+
+def test_exponential_iso():    
+    fdata, fbvals, fbvecs = dpd.get_data()
+    data_dti = nib.load(fdata).get_data()
+    gtab_dti = grad.gradient_table(fbvals, fbvecs)
+    data_dsi, gtab_dsi = dpd.dsi_voxels()
+    for data, gtab in zip([data_dsi, data_dti], [gtab_dsi, gtab_dti]):
+        sfmodel = sfm.SparseFascicleModel(gtab,
+                                        isotropic=sfm.ExponentialIsotropicModel)
```

Comment:
This one's a bit tough: If I properly indent, the line goes to more than 80 characters. I preferred to go this way, because it seems to be less than an eye-sore than having the line wrap down. I could also break it: 

    istropic=
    sfm.ExponentialIsotropicModel

What do you think?